### PR TITLE
Updated secret management page

### PR DESCRIPTION
**Pull request by Jay-umbr**
Added an explanation on how to overwrite keys in appSettings.json using secrets.

### CODE REVIEW

File: umbraco-cloud/set-up/project-settings/secrets-management.md [hunk 1]
```diff
@@ -68,6 +68,16 @@ You can use the method, getting it from the System namespace in .NET as below:
 
 `_secretMessage = Environment.GetEnvironmentVariable("SecretMessage");`
 
+Secrets can also be used to override AppSettings defined in appsettings.json files.
```
Comment (sofietoft) [inline umbraco-cloud/set-up/project-settings/secrets-management.md:71]:
```suggestion
Secrets can also be used to override AppSettings defined in `appsettings.json` files.
```

File: umbraco-cloud/set-up/project-settings/secrets-management.md [hunk 2]
```diff
@@ -68,6 +68,16 @@ You can use the method, getting it from the System namespace in .NET as below:
...
+
+In order for this to work, when adding the secret, the Key value should be all the settings' names joined by double underscores. 
+
+For example, to change the Serilog's default options under "Serilog" > "MinimumLevel" > "Default" key, the Secret key would look like this:
```
Comment (sofietoft) [inline umbraco-cloud/set-up/project-settings/secrets-management.md:75]: Is this a reference a setting in appsettings.json?

```suggestion
For example, to change the Serilog's default options under `Serilog:MinimumLevel:Default`, the Secret key would look like this:
```

File: umbraco-cloud/set-up/project-settings/secrets-management.md [hunk 3]
```diff
@@ -68,6 +68,16 @@ You can use the method, getting it from the System namespace in .NET as below:
...
+
+`Serilog__MinimumLevel__Default`
+
+The value defined in appsettings.json file will be overwritten with the Cloud Secret's value.
```
Comment (sofietoft) [inline umbraco-cloud/set-up/project-settings/secrets-management.md:79]:
```suggestion
The value defined in `appsettings.json` file will be overwritten with the Cloud Secret's value.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR 💪 

I've made a couple of comments.

File: umbraco-cloud/set-up/project-settings/secrets-management.md (hunk 1 above)
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/secrets-management.md:71]:
```suggestion
Secrets can also be used to override AppSettings defined in `appsettings.json files`.
```

File: umbraco-cloud/set-up/project-settings/secrets-management.md (hunk 3 above)
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/secrets-management.md:79]:
```suggestion
The value defined in `appsettings.json` file will be overwritten with the Cloud Secret's value.
```

File: umbraco-cloud/set-up/project-settings/secrets-management.md (hunk 2 above)
Comment (Jay-umbr) [reply umbraco-cloud/set-up/project-settings/secrets-management.md:75]: Yes, that is right

**Pull request merged**